src/components/course-settings/remove-teacher.cjsx

```diff
@@ -25,10 +24,11 @@ module.exports = React.createClass
       .value()
 
   goToDashboard: ->
-    @context.router.transitionTo('dashboard', {courseId: @props.courseId})
+    TeacherRosterStore.once 'deleted', ->
+      window.location.href = '/dashboard/'
```

Comment:
Does `@context.router.transitionTo('dashboard', {courseId: @props.courseId})` no longer work in `TeacherRosterStore.once 'deleted', =>` ?